src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java

```diff
@@ -42,4 +42,18 @@ public static boolean isBlank(int... codePoints) {
                 .allMatch(Character::isWhitespace);
     }
 
+    /**
+     * Checks if the given code point array contains only whitespace up to specified index.
+     *
+     * @param codePoints
+     *            array of Unicode code point of string to check
+     * @param index
+     *            index to check up to (exclusive)
+     * @return true if all code points preceding given index are whitespace
+     */
+    public static boolean hasWhitespaceBefore(int index, int... codePoints) {
+        return Arrays.stream(codePoints, 0, index)
+                .allMatch(codePoint -> Character.isWhitespace(Character.toChars(codePoint)[0]));
```

Comment:
Please use method [isWhitespace](https://docs.oracle.com/javase/8/docs/api/java/lang/Character.html#isWhitespace-int-). No need to convert it to char[] then back to int.

Reply:
done